README.md

```diff
@@ -80,7 +80,7 @@ _Disclaimer: Package manifests are maintained by the community, not Marp team._
 
 <!-- For contributors: This section describes only package managers that Marp manifest has been actively maintained. Each tools are following update within a few days of the latest CLI update. -->
 
-#### macOS: **[Homebrew](https://brew.sh/)**
+#### macOS for Apple Silicon architecture: **[Homebrew](https://brew.sh/)**
```

Comment:
Misleading: Homebrew package also supports Intel Mac architecture.

Homebrew will install Marp CLI through npm so it works on any architecture. #620 indicates that it is necessary to mention the architecture just only for the standalone binary.

Reply:
Thank you, I moved the description to the standalone section.